src/storage/copy_arrow/copy_structures_arrow.cpp

```diff
@@ -317,7 +317,61 @@ std::unique_ptr<Value> CopyStructuresArrow::getArrowList(std::string& l, int64_t
             BufferPoolConstants::DEFAULT_PAGE_SIZE, numBytesOfOverflow));
     }
     return make_unique<Value>(
-        DataType(LIST, std::make_unique<DataType>(childDataType)), std::move(values));
+        DataType(VAR_LIST, std::make_unique<DataType>(childDataType)), std::move(values));
+}
+
+std::unique_ptr<uint8_t[]> CopyStructuresArrow::getArrowFixedList(std::string& l, int64_t from,
+    int64_t to, const DataType& dataType, CopyDescription& copyDescription) {
+    assert(dataType.typeID == common::FIXED_LIST);
+    auto childDataType = *dataType.childType;
+
+    char delimiter = copyDescription.csvReaderConfig->delimiter;
```

Comment:
Let's reuse the split part here from `getArrowVarList`. You can make this as a new function.

Reply:
created a new function here.